scrapy/downloadermiddlewares/decompression.py

```diff
@@ -7,14 +7,9 @@
 import zipfile
 import tarfile
 import logging
-from tempfile import mktemp
-
 import six
-
-try:
-    from cStringIO import StringIO as BytesIO
-except ImportError:
-    from io import BytesIO
+from six import BytesIO
```

Comment:
This is not the same as before, because six defines BytesIO as StringIO.StringIO, not cStringIO.StringIO, which may cause a big performance hit.

Reply:
@kmike can you check my first commit. We can replace try/except with if/else.